docs/creating.md

```diff
@@ -75,27 +75,58 @@ their usage.
 
 In addition to
 [all the features Jinja supports](https://jinja.palletsprojects.com/en/3.1.x/templates/),
-Copier includes:
-
--   All functions and filters from
-    [jinja2-ansible-filters](https://gitlab.com/dreamer-labs/libraries/jinja2-ansible-filters/).
-
-    -   This includes the `to_nice_yaml` filter, which is used extensively in our
-        context.
-
--   `_copier_answers` includes the current answers dict, but slightly modified to make
-    it suitable to [autoupdate your project safely][the-copier-answersyml-file]:
-    -   It doesn't contain secret answers.
-    -   It doesn't contain any data that is not easy to render to JSON or YAML.
-    -   It contains special keys like `_commit` and `_src_path`, indicating how the last
-        template update was done.
--   `_copier_conf` includes a representation of the current Copier
-    <!-- prettier-ignore -->
-    [Worker][copier.main.Worker] object, also slightly modified:
-    -   It only contains JSON-serializable data.
-    -   You can serialize it with `{{ _copier_conf|to_json }}`.
-    -   ⚠️ It contains secret answers inside its `.data` key.
-    -   Modifying it doesn't alter the current rendering configuration.
-    -   It contains the current commit hash from the template in
-        `{{ _copier_conf.vcs_ref_hash }}`.
-    -   Contains Operating System-specific directory separator under `sep` key.
+Copier provides all functions and filters from
+[jinja2-ansible-filters](https://gitlab.com/dreamer-labs/libraries/jinja2-ansible-filters/).
+This includes the `to_nice_yaml` filter, which is used extensively in our context.
+
+## Variables (global)
+
+The following variables are always available in Jinja templates:
+
+### `_copier_answers`
+
+`_copier_answers` includes the current answers dict, but slightly modified to make it
+suitable to [autoupdate your project safely][the-copier-answersyml-file]:
+
+-   It doesn't contain secret answers.
+-   It doesn't contain any data that is not easy to render to JSON or YAML.
+-   It contains special keys like `_commit` and `_src_path`, indicating how the last
+    template update was done.
+
+### `_copier_conf`
+
+`_copier_conf` includes a representation of the current Copier
+[Worker][copier.main.Worker] object, also slightly modified:
+
+-   It only contains JSON-serializable data.
+-   You can serialize it with `{{ _copier_conf|to_json }}`.
+-   ⚠️ It contains secret answers inside its `.data` key.
+-   Modifying it doesn't alter the current rendering configuration.
+
+Furthermore, the following keys are added:
+
+#### `_copier_conf.os`
```

Comment:
I find these subheadings in the TOC sidebar a bit verbose and perhaps not necessary because they are also indented. How about replacing them by only the key names and adding a custom ID attribute to have a proper heading anchor?

```suggestion
#### `os` { #_copier_conf.os }
```

For this, we'd need to [add the `attr_list` Markdown extension in `mkdocs.yml`](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#attribute-lists).